extensions/lifecycle/bucketProcessor/LifecycleBucketProcessor.js

```diff
@@ -236,14 +236,29 @@ class LifecycleBucketProcessor {
      * @return {Boolean} Whether the config should be processed
      */
     _shouldProcessConfig(config) {
-        if (config.Rules.length === 0) {
-            this._log.debug('bucket lifecycle config has no rules to process', {
-                config,
-            });
-            return false;
+        const rulesEnabled = config.Rules.some(rule => {
+            if (rule.Status === 'Disabled') {
+                return false;
+            }
+            if (rule.Transition &&
+                this._pausedLocations.has(rule.Transition.StorageClass)) {
+                return false;
+            }
+            if (rule.NoncurrentVersionTransition &&
+                this._pausedLocations.has(rule.NoncurrentVersionTransition.StorageClass)) {
+                return false;
+            }
```

Comment:
i wonder if it is not possible to have multiple "kinds" (transition, ncvTransiton, Expiration...) in the same rule : and thus we may have one transition to paused location but another one to non-paused location...